src/psyclone/tests/psyir/transformations/intrinsics/sign2code_trans_test.py

```diff
@@ -33,11 +33,11 @@
 # -----------------------------------------------------------------------------
 # Author R. W. Ford, STFC Daresbury Lab
 
-'''Module containing tests for the nemo abs transformation.'''
+'''Module containing tests for the abs2code transformation.'''
```

Comment:
s/abs2code/sign2code/

Reply:
Oops. I didn't notice it was wrong in the first place and just changed it without thinking.